### PR TITLE
Backport 17015, 16397, 15459

### PR DESCRIPTION
**Pull request by humbleDasher**

### CODE REVIEW

File: doc/release-process.md [hunk 1]
```diff
@@ -276,3 +276,23 @@ Note: check that SHA256SUMS itself doesn't end up in SHA256SUMS, which is a spur
...
+
+### Additional information
+
+#### How to calculate `m_assumed_blockchain_size` and `m_assumed_chain_state_size`
+
+Both variables are used as a guideline for how much space the user needs on their drive in total, not just strictly for the blockchain.
+Note that all values should be taken from a **fully synced** node and have an overhead of 5-10% added on top of its base value.
+
+To calculate `m_assumed_blockchain_size`:
+- For `mainnet` -> Take the size of the Bitcoin data directory, excluding `/regtest` and `/testnet3` directories.
```
Comment (PastaPastaPasta) [inline doc/release-process.md:288]: Dashify (and squash into original commit)

Comment (humbleDasher) [reply doc/release-process.md:288]: Got it!!! Thank you!!!!

**Review by PastaPastaPasta — APPROVED** [review]
LGTM

utACK for merging via merge commit

**Review by UdjinM6 — APPROVED** [review]
utACK

**Pull request merged**